isaric/parsers/isaric_washington.toml

```diff
@@ -255,274 +264,144 @@
-    fields = [
-      { field = "avpu_score_v2___1", values = { 1 = "Alert" } },
-      { field = "avpu_score_v2___2", values = { 1 = "Verbal" } },
-      { field = "avpu_score_v2___3", values = { 1 = "Pain" } },
-      { field = "avpu_score_v2___4", values = { 1 = "Unresponsive" } },
-    ]
-
-[[observation]]
-  name = "diastolic_blood_pressure_mmHg"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "$hospital_day_core_e1",
-  ] } }
-  phase = "study"
-  value = { field = "dbp_echo" }
-
-[[observation]]
-  name = "pao2_mmHg"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "$hospital_day_core_e1",
-  ] } }
-  phase = "study"
-  value = { field = "pao2_at_time_of_fio2_above" }
-  context = ["Arterial", "Same blood gas record"]
-
-[[observation]]
-  name = "pco2_mmHg"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "$hospital_day_core_e1",
-  ] } }
-  phase = "study"
-  value = { field = "pco2_from_same_blood_gas_r" }
-  context = ["Same blood gas record"]
-
-[[observation]]
-  name = "pH"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "hospital_day_core_e1",
-  ] } }
-  phase = "study"
-  value = { field = "ph_from_same_blood_gas_rec" }
-  context = ["Same blood gas record"]
-
-[[observation]]
-  name = "pao2_mmHg"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "$hospital_day_core_e2",
-  ] } }
-  phase = "study"
-  value = { field = "pao2_lowest_plods2", source_unit = "mmHg" }
-  context = ["Lowest daily reading"]
-
-[[observation]]
-  name = "glasgow_coma_score"
-  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
-    "$hospital_day_core_e2",
-  ] } }
-  phase = "study"
-  value = { field = "gcs_lowest_plods2" }
-  context = ["Lowest daily score"]
-
 [[observation]]
   name = "abdominal_pain"
   date = { field = "ADMIT DATE" }
   phase = "admission"
-  is_present = { field = "signs_and_symptoms_at_hosp___1", ref = "Y/N" }
+  is_present = { field = "signs_and_symptoms_at_hosp___1", values = { 1 = true } }
 
 [[observation]]
-  name = "altered_consciousness_confusion"
+  name = "confusion"
   date = { field = "ADMIT DATE" }
   phase = "admission"
-  is_present = { field = "signs_and_symptoms_at_hosp___8", ref = "Y/N" }
+  is_present = { field = "signs_and_symptoms_at_hosp___8", values = { 1 = true } }
 
 [[observation]]
   name = "anorexia"
   date = { field = "ADMIT DATE" }
   phase = "admission"
-  is_present = { field = "signs_and_symptoms_at_hosp___3", ref = "Y/N" }
+  is_present = { field = "signs_and_symptoms_at_hosp___3", values = { 1 = true } }
 
 [[observation]]
   name = "chest_pain"
   date = { field = "ADMIT DATE" }
   phase = "admission"
-  is_present = { field = "signs_and_symptoms_at_hosp___6", ref = "Y/N" }
+  is_present = { field = "signs_and_symptoms_at_hosp___6", values = { 1 = true } }
 
 [[observation]]
   name = "confusion"
```

Comment:
There is another "confusion" above with the same labels and values .. we should remove this.

Reply:
good catch, thanks @ekamau !